### DISCUSSION TITLE
Rollout of an entire ApplicationSet?

### DISCUSSION THREAD

**Discussion by zswanson** [argoproj/argo-rollouts#1562] [Q&A] [answered]
Is it possible to use Rollout with an ApplicationSet, creating a blue/green workflow for an entire stack of Argo Applications?

**Comment by harikrongali** [comment, marked as answer]
interesting requirement and I dont think Rollout supports it. I will include @jessesuen / @alexmt to provide some insight

**Reply by zswanson** [reply]
My company currently does a full 'system' blue-green on ec2, but we're migrating to k8s and there is some desire to keep this same pattern. I'm not in favor of keeping it that way, so just doing my checks to what sort of options exist.

**Reply by jessesuen** [reply]
No, Argo Rollouts is at an individual workload level. So it doesn't work at an aggregate application level.
